composer.json

```diff
@@ -87,6 +87,7 @@
         "guzzlehttp/guzzle": "^7.2",
         "league/flysystem-aws-s3-v3": "^3.0",
         "league/flysystem-ftp": "^3.0",
+        "league/flysystem-read-only": "^3.3",
```

Comment:
Does this need adding to the `suggest` block?

Reply:
I've added the suggests entry.